app/views/claims/bank_details.html.erb

```diff
@@ -0,0 +1,42 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
+      <%= render("shared/error_summary", instance: current_claim) if current_claim.errors.any? %>
+
+      <%= form_for current_claim, url: claim_path do |form| %>
+        <fieldset class="govuk-fieldset">
+          <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
+            <h1 class="govuk-fieldset__heading">
+              Your bank account details
+            </h1>
+          </legend>
+
+          <p class="govuk-body govuk-hint">
+            We need to collect this information to pay you.
+          </p>
+
+          <div class="govuk-form-group">
+            <%= form.label :bank_sort_code, "Sort code", class: "govuk-label" %>
+            <span  id="sort-code-hint" class="govuk-hint">For example: 44 00 26</span>
+            <%= form.text_field :bank_sort_code,
+              class: "govuk-input govuk-!-width-one-quarter",
+              "aria-describedby" => "sort-code-hint"
+            %>
+          </div>
+
+          <div class="govuk-form-group">
+            <%= form.label :bank_account_number, "Account number", class: "govuk-label" %>
+            <span  id="account-number-hint" class="govuk-hint">For example: 70 87 24 90</span>
+            <%= form.text_field :bank_account_number,
+              class: "govuk-input govuk-input--width-20",
+              "aria-describedby" => "account-number-hint"
+            %>
+          </div>
+
+        </fieldset>
+        <button type="submit" class="govuk-button">
+          Continue
```

Comment:
We could use `<%= form.submit "Continue", class: "govuk-button" %>` so we're consistent with the other pages

Reply:
👍